azext_iot/tests/central/test_iot_central_int.py

```diff
@@ -410,6 +412,32 @@ def test_central_device_template_methods_CRD(self):
         assert deleted_dev_temp_count == start_dev_temp_count
         assert template_id not in deleted_device_template_list.keys()
 
+    @pytest.mark.skipif(
+        not STORAGE_CSTRING or not STORAGE_CONTAINER,
+        reason="empty azext_iot_central_storage_cstring or azext_iot_central_storage_container env var",
+    )
+    def test_central_fileupload_methods_CRD(self):
+        file_upload = self._create_fileupload()
+        self._wait_for_storage_configured()
+        command = self._appendOptionalArgsToCommand(
```

Comment:
Can't this be abstracted so that the tester does not have to remember to call _appendOptionalArgsToCommand with token and dns_suffix before calling the cmd? This may be consideration for possible refactorization.

Reply:
We'll get on separate PR